packages/@react-spectrum/s2/style/properties.json

```diff
@@ -0,0 +1,2422 @@
+    "backgroundImage": "__a",
+    "filter": "__b",
+    "float": "__c",
+    "gridAutoFlow": "__d",
+    "justifyItems": "__e",
+    "justifySelf": "__f",
+    "stroke": "__g",
+    "textDecoration": "__h",
+    "animationFillMode": "__i",
+    "resize": "__j",
+    "willChange": "__k",
+    "animationDirection": "__l",
+    "backgroundClip": "__m",
+    "breakInside": "__n",
+    "scale": "__o",
+    "scrollSnapAlign": "__p",
+    "strokeWidth": "__q",
+    "textTransform": "__r",
+    "textWrap": "__s",
+    "listStyleType": "__t",
+    "backgroundAttachment": "__u",
+    "backgroundOrigin": "__v",
+    "backgroundSize": "__w",
+    "hyphens": "__x",
+    "overscrollBehaviorX": "__y",
+    "overscrollBehaviorY": "__z",
+    "wordBreak": "__A",
+    "zIndex": "__B",
+    "boxDecorationBreak": "__C",
+    "textUnderlineOffset": "__D",
+    "animationDuration": "__E",
+    "appearance": "__F",
+    "borderCollapse": "__G",
+    "captionSide": "__H",
+    "listStylePosition": "__I",
+    "scrollBehavior": "__J",
+    "scrollSnapStop": "__K",
+    "tableLayout": "__L",
+    "transform": "__M",
+    "-webkit-box-orient": "__N",
+    "-webkit-line-clamp": "__O",
+    "animationName": "__P",
+    "gridAutoRows": "__Q",
+    "overflow": "__R",
+    "transitionDelay": "__S",
+    "animationIterationCount": "__T"
+  },
+  "values": {
+    "fontSize": {
+      "0.875rem": "a",
+      "1.0625rem": "b",
+      "1.375rem": "c",
+      "1.1875rem": "d",
+      "1rem": "e",
+      "0.75rem": "f",
+      "0.9375rem": "g",
+      "1.125rem": "h",
+      "0.6875rem": "i",
+      "0.8125rem": "j",
+      "1.25rem": "k",
+      "1.5rem": "l",
+      "1.6875rem": "m",
+      "1.5625rem": "n",
+      "1.9375rem": "o",
+      "1.75rem": "p",
+      "2.125rem": "q",
+      "2.25rem": "r",
+      "2.75rem": "s",
+      "2.8125rem": "t",
+      "3.4375rem": "u",
+      "3.625rem": "v",
+      "4.375rem": "w"
+    },
+    "fontFamily": {
+      "adobe-clean-han-japanese, 'Hiragino Kaku Gothic ProN', 'ヒラギノ角ゴ ProN W3', Osaka, YuGothic, 'Yu Gothic', 'メイリオ', Meiryo, 'ＭＳ Ｐゴシック', 'MS PGothic', sans-serif": "a",
+      "adobe-clean-han-korean, source-han-korean, 'Malgun Gothic', 'Apple Gothic', sans-serif": "b",
+      "adobe-clean-han-simplified-c, source-han-simplified-c, 'SimSun', 'Heiti SC Light', sans-serif": "c",
+      "adobe-clean-han-traditional, source-han-traditional, 'MingLiu', 'Heiti TC Light', sans-serif": "d",
+      "adobe-clean-han-traditional, source-han-traditional, 'MingLiu', 'Microsoft JhengHei UI', 'Microsoft JhengHei', 'Heiti TC Light', sans-serif": "e",
+      "myriad-arabic, ui-sans-serif, system-ui, sans-serif": "f",
+      "myriad-hebrew, ui-sans-serif, system-ui, sans-serif": "g",
+      "adobe-clean-variable, adobe-clean, ui-sans-serif, system-ui, sans-serif": "h",
+      "adobe-clean-serif, \"Source Serif\", Georgia, serif": "i",
+      "source-code-pro, \"Source Code Pro\", Monaco, monospace": "j"
+    },
+    "backgroundColor": {
```

Comment:
Color properties currently do not include all possible values defined in the theme, because there are a lot of them. Right now we only include values currently being used in S2 components. If applications use additional colors then those will currently be hashed. We can additional shorter color mappings in the future but I want to be sure that things have stabilized before assigning these names, since these short names cannot be reused in the future.